src/book.cpp

```diff
@@ -42,72 +41,44 @@ int getBook(char* booktitle)
 
 void createBooks()
 {
-	node_t* node;
 	string_t* name = NULL;
-	bool unsorted;
 	int i = 0;
 
 	//TODO: Read the books/ inventory for all *.txt files.
 	//TODO: Then create a book for each file there and add it to a books array.
-	discoveredbooks = directoryContents("books/");
-	if (discoveredbooks)
+	auto discoveredbooks = directoryContents("books/");
```

Comment:
> Barony uses C++11 features like auto

For the record, the coding style guidelines prefer avoidance of auto.
(It seems Barony is in violation of many things...)